js/reducers/windows.js

```diff
@@ -15,8 +15,8 @@ const defaultWindowsState = {
   focused: WINDOWS.MAIN,
   equalizer: true,
   playlist: true,
-  // openGenWindows: ["AVS_WINDOW"]
-  openGenWindows: [],
+  openGenWindows: ["MILKDROP_WINDOW"],
```

Comment:
I've added https://github.com/captbaritone/webamp/commit/569a6819f0a9740f18e80d12ea5a006c250fca8c so this change can be omitted.